tchannel-benchmark/src/main/java/com/uber/tchannel/benchmarks/PingPongServerBenchmark.java

```diff
@@ -177,5 +199,14 @@ public int actualQPS() {
         public int errorQPS() {
             return errorQPS.get();
         }
+        public int timeoutQPS() {
```

Comment:
This is used by the benchmark platform. It uses getter to print out the QPS result.